server/sonar-server/src/main/java/org/sonar/server/startup/RenameIssueWidgets.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * SonarQube, open source software quality management tool.
+ * Copyright (C) 2008-2014 SonarSource
+ * mailto:contact AT sonarsource DOT com
+ *
+ * SonarQube is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * SonarQube is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.server.startup;
+
+import com.google.common.collect.Lists;
+import org.picocontainer.Startable;
+import org.sonar.api.utils.log.Loggers;
+import org.sonar.core.dashboard.WidgetDto;
+import org.sonar.core.dashboard.WidgetPropertyDto;
+import org.sonar.core.issue.db.IssueFilterDto;
+import org.sonar.core.persistence.DbSession;
+import org.sonar.core.persistence.MyBatis;
+import org.sonar.core.template.LoadedTemplateDto;
+import org.sonar.server.db.DbClient;
+
+import java.util.List;
+
+public class RenameIssueWidgets implements Startable {
+
+  private static final String PROJECT_ISSUE_FILTER_WIDGET_KEY = "project_issue_filter";
+  private static final String FILTER_PROPERTY = "filter";
+  private static final String DISTRIBUTION_AXIS_PROPERTY = "distributionAxis";
+
+  private final DbClient dbClient;
+
+  public RenameIssueWidgets(DbClient dbClient) {
+    this.dbClient = dbClient;
+  }
+
+  @Override
+  public void start() {
+    DbSession session = dbClient.openSession(false);
+
+    try {
+      if (dbClient.loadedTemplateDao().countByTypeAndKey(LoadedTemplateDto.ONE_SHOT_TASK_TYPE, getClass().getSimpleName()) != 0) {
```

Comment:
I'm not sure using the class name to persist in db in a good idea : it the classe name changed in next version, the task will be executed again.
I think using a constant would be better.